File: content/learn/arrays-easy/01-largest-element/solution.js
function largest(a) {
  let best = a[0];
  for (let i = 1; i < a.length; i++) {
    if (a[i] > best) best = a[i];
  }
  return best;
}

const n = parseInt(readline(), 10);
const a = readline().split(" ").map(Number);
print(largest(a));

File: content/learn/arrays-easy/01-largest-element/solution.cpp
#include <iostream>
#include <vector>
using namespace std;

int largest(const vector<int>& a) {
    int best = a[0];
    for (size_t i = 1; i < a.size(); i++) {
        if (a[i] > best) best = a[i];
    }
    return best;
}

int main() {
    int n;
    cin >> n;
    vector<int> a(n);
    for (int i = 0; i < n; i++) cin >> a[i];
    cout << largest(a) << "\n";
    return 0;
}
